test/integration/users.test.js

```diff
@@ -1229,4 +1229,31 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("PATCH /users/remove-tokens", function () {
+    it("should remove all the users with token field", function (done) {
+      before(async function () {
+        await addOrUpdate(userData[0]);
+        await addOrUpdate(userData[1]);
+        await addOrUpdate(userData[2]);
+        await addOrUpdate(userData[3]);
+      });
+      after(async function () {
+        await cleanDb();
+      });
+
+      chai
+        .request(app)
+        .patch("/users/remove-tokens")
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body.message).to.be.equal("Github Token removed from all users!");
```

Comment:
I think we should run the `fetchUsersWithToken` model function after this and assert if the users array returned is empty or not to check if the migration was successful

Reply:
@shubham-y i dont think we can do await inside chai assertion

Reply:
> @shubham-y i dont think we can do await inside chai assertion

We can dot it. Check these for reference

https://github.com/Real-Dev-Squad/website-backend/blob/0c370f9aa903e8e4d6131e9a9a9a66eb45c690d4/test/integration/auth.test.js#L39-L41
https://github.com/Real-Dev-Squad/website-backend/blob/0c370f9aa903e8e4d6131e9a9a9a66eb45c690d4/test/unit/services/discordService.test.js#L24-L30

Reply:
@shubham-y I have made changes to the tests, please check